week-4/Homework/mandatory/2-writers.js

```diff
@@ -53,19 +53,36 @@ Exercise 1:
   insert corresponding values to the place holder that are indicated in courle braces:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
-
+let bioList = writers
+.forEach(n=> console.log(
+  "Hi, my name is " +
+   n.firstName + " " +
+     n.lastName + ". I am " +
+      n.age + " years old, and work as a " +
+        n.occupation + "."));
 
 
 /*
 Exercise 2:
   Only `console.log()` the writers who are in their 40s with us anymore with the sentence:(meaning between 40 and 49)
   "Writer {firstName} {lastName} died at {age} years old."
 */
-
+let deathList = writers.filter(a=> a.age >= 40 && a.age < 50 && a.alive === true)
+.forEach(n=> console.log(
+  "Writer " +
+   n.firstName + " " +
+     n.lastName + " died at " +
+      n.age + " years old."));

```

Comment:
Hhhhh deadList that funny. You don't need of this variable. I think you haven't made fetch upstream because the instruction is wrong here. Do fetch again.